.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request 💡
+description: Have any new idea or new feature ? Please suggest!
+title: "[Feature] <write a small description here>"
+labels: ["enhancement", "goal: addition"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of any alternative solution or features you've considered.
```

Comment:
I think there should be two `textarea`, one for the feature description and other one for the alternative solutions of things they have tried.